Fixing PaaS cloud service static IP address table

kwill-MSFT opened the pull request:
PaaS Cloud Services cannot have static DIPs or multiple DIPs.  This article was correct when originally published, but the change https://github.com/MicrosoftDocs/azure-docs/commit/a1e93b2f8cebec326ed425bc9fc0070e97deb2f9#diff-f9782756ab1ca3f72ee0b08c2282b2b9 changed several tables and introduced some errors for PaaS cloud service role instances in respect to private IP addresses.

azurecla commented:
Hi __@clpolet__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (clpolet). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

kwill-MSFT committed:
Fixing PaaS cloud service static IP address table

PaaS Cloud Services cannot have static DIPs or multiple DIPs.  This article was correct when originally published, but the change https://github.com/MicrosoftDocs/azure-docs/commit/a1e93b2f8cebec326ed425bc9fc0070e97deb2f9#diff-f9782756ab1ca3f72ee0b08c2282b2b9 changed several tables and introduced some errors for PaaS cloud service role instances in respect to private IP addresses.
